source/2020-05-29-the-ember-times-issue-150.md

```diff
@@ -57,25 +59,25 @@ What's in store for the future? The [project](https://github.com/ember-learn/emb
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [Test Directory RFC Final Comment Period Closing ⏳](https://github.com/emberjs/rfcs/pull/575)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+RFC final comment period closing alert 🚨!
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+The ✨awesome ✨ RFC by [Mehul Kar (@mehulkar)](https://github.com/mehulkar) about changing the name of an Ember app's test directories to be more in sync with naming provided in the official guides has entered the last week of final comment period. 
+
+The RFC proposes that this unification could make it easier to have a shared language between tests, and would reduce confusion for newer users when using Ember CLI.
+
+If you're interested in the subject and have opinions you want to voice, now would be a good time to head on over to the [RFC](https://github.com/emberjs/rfcs/pull/575) and provide some feedback. 
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [New Router Service Refresh RFC ♻️](https://github.com/emberjs/rfcs/pull/631)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+There's a cool 😎 new RFC by [Gaurav Munjal (@Gaurav0)](https://github.com/Gaurav0) regarding adding a `refresh` method to the router service.
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+Note this `refresh` method would be different than the currently available `Route.prototype.refresh` method that refreshes only the refreshes whatever route you call it on and all routes “below” it in the route hierarchy. This newly proposesd method would refresh all routes.
```

Comment:
I'm not familiar with the RFC so I wasn't sure what you meant by "only the refreshes whatever route ... in the route hierarchy." Is it possible to rephrase this part of the sentence?

Can we fix the spelling from `proposesd` to `proposed`?

Reply:
thanks, taking a looksie at the RFC re: your comment  👀

Reply:
Thanks for getting the adjustments worked out @amyrlam and @ijlee2 !

Reply:
👍 really appreciate the writeups, thank you!